lms/views/lti/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""Baseclass for tasks common to LTI Launch and the Configure Module Item views."""
+
+from lms.services import HAPIError
+from lms.views.helpers import via_url
+
+
+class LTIViewBaseClass:
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+
+        # Configure the front-end mini-app to run.
+        self.context.js_config.update({"mode": "basic-lti-launch"})
+
+        # Add config used by frontend to call `record_submission` API.
+        params = self.request.params
+        if (
```

Comment:
What is this `if` actually testing as a description? It would be nice to tuck this into a named private function